pkg/bootstrap/docker/openshift/logging.go

```diff
@@ -0,0 +1,140 @@
+package openshift
+
+import (
+	"bytes"
+	"fmt"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kerrors "k8s.io/kubernetes/pkg/util/errors"
+
+	"github.com/openshift/origin/pkg/bootstrap/docker/errors"
+	"github.com/openshift/origin/pkg/client"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	configcmd "github.com/openshift/origin/pkg/config/cmd"
+	genappcmd "github.com/openshift/origin/pkg/generate/app/cmd"
+)
+
+const (
+	loggingNamespace               = "logging"
+	svcKibana                      = "kibana-logging"
+	loggingDeployerAccountTemplate = "logging-deployer-account-template"
+	loggingDeployerTemplate        = "logging-deployer-template"
+)
+
+func instantiateTemplate(client client.Interface, mapper configcmd.Mapper, templateNamespace, templateName, targetNamespace string, params map[string]string) error {
```

Comment:
@bparees so I looked into merging this code with the one from the admission controller, but there's very little that we can actually share. Basically we could share the call to the bulk create. However, even then, you have a custom After handler in that code that is not necessarily what I want in this case.

https://github.com/openshift/origin/blob/master/pkg/build/admission/jenkinsbootstrapper/admission.go#L112